config.yaml

```diff
@@ -183,6 +183,13 @@ options:
 
           This requires configuration on the S3 side, as well as a suitable Mattermost licence.
         default: false
+    s3_tls:
```

Comment:
Is there any other way to do this? It seems to open up a path to mistakes of accidentally disabling this

Reply:
I removed the option